MAINTAINERS.yml

```diff
@@ -2859,17 +2859,21 @@ RISCV arch:
     - carlocaione
     - npitre
     - ycsin
+    - pfarwsi
+    - kanakshilledar
```

Comment:
It will take a bit of time before you can become arch maintainers as pointed out by @fkokosinski 

Review roles here 

https://docs.zephyrproject.org/latest/project/project_roles.html

```suggestion
```

Reply:
Ok i have incorporated the changes @fkokosinski @cfriedt